### PR TITLE
[10.0] [FIX] hr_attendance_rfid UserError catch on system + test

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
FF if #469

### CODE REVIEW

File: hr_attendance_rfid/models/hr_employee.py [hunk 1]
```diff
@@ -66,7 +66,5 @@ def register_attendance(self, card_code):
                 res['error_message'] = msg
                 return res
         except Exception as e:
-            msg = e.message
-            _logger.error(msg)
```
Comment (pedrobaeza) [inline hr_attendance_rfid/models/hr_employee.py:70]: I assume you are removing this for having tests correct, but what you can do it's a `mute_logger` in the test for that purpose.

**Comment by hveficent** [comment]
Aren't UserError exceptions running automatically in logs?

File: hr_attendance_rfid/models/hr_employee.py (hunk 1 above)
Comment (HviorForgeFlow) [reply hr_attendance_rfid/models/hr_employee.py:70]: Aren't UserError exceptions running automatically in logs?

Comment (pedrobaeza) [reply hr_attendance_rfid/models/hr_employee.py:70]: No, the Travis status is determined if there's an `ERROR` text parsed in the log.

**Review by rafaelbn — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**